librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -98,7 +98,7 @@
 #define AND(x, y) rz_il_op_new_bool_and(x, y)
 #define OR(x, y)  rz_il_op_new_bool_or(x, y)
 
-#define FNEQ(flx, fly) OR(FORDER(flx, fly), FORDER(DUP(flx), DUP(fly)))
+#define FNEQ(flx, fly) OR(FORDER(flx, fly), FORDER(DUP(fly), DUP(flx)))
```

Comment:
i'm not sure if these should have `DUP` in there.

Reply:
Sure? `flx` and `fly` are used twice. Once in each `FORDER`. Each `FORDER` will free it's version `flx/y`. So the `DUP` becomes necessary to prevent double frees.

Reply:
oh i misread this. yeah, then is correct